HCK20_2024_Remote/README.md

```diff
@@ -0,0 +1,90 @@
+<img alt="NWB 2024 User/Developer Days Banner" src="images/logo_brain_text_white_hor.png">
+
+# NWB User Days 2024
```

Comment:
I know we have a long history of calling this event the "NWB User Days" but this does not really convey what the workshop is about, especially now that it is centered around converting data to NWB. What about "NWB Data Conversion Workshop" or "NWB User Days 2024: Convert Your Own Data"? @stephprince @oruebel @bendichter

Reply:
How about including GUIDE in the subheading? E.g. "NWB User Days: A  guide to NWB conversion and adoption"

Reply:
I'm in support of changing the name. I like: "NWB Data Conversion Workshop"